.github/workflows/reliably.yaml

```diff
@@ -9,8 +9,11 @@ jobs:
     steps:
       - name: 'Checkout source code'
         uses: actions/checkout@v2
-      - name: 'Run Reliably'
-        uses: reliablyhq/gh-action@main
+      - name: 'Run Reliably Scan'
+        uses: reliablyhq/actions/scan@v1
+        env:
+          RELIABLY_TOKEN: ${{ secrets.RELIABLY_TOKEN }}
+          RELIABLY_ORG: ${{ secrets.RELIABLY_ORG }}
         continue-on-error: true
```

Comment:
I think what we want is to remove continue-on-error - this was why we kept missing the reliably auth step

Thoughts @Lawouach ? I've no idea why this was set in the first place

Reply:
The flag allows to kinds of behavior:

* Fail the build
* Warn about issues
The latter can handy when you're not interested to block a build just because of scanning issues. It's limited (and should likely be tied to level of errors returned during scan)

Reply:
Right, but the following step will always fail no?

Seems like it's redundant in this case as we have a dependency on this block passing anyway